src/main/java/org/gridsuite/directory/server/DirectoryService.java

```diff
@@ -88,7 +84,7 @@ public DirectoryService(
                     deleteElement(studyUuid, userId).subscribe();
                 }
                 boolean isPrivate = isPrivateForNotification(parentUuid, isPrivateDirectory(studyUuid));
-                emitDirectoryChanged(parentUuid, userId, error, isPrivate, parentUuid == null, NotificationType.UPDATE_DIRECTORY);
+                emitDirectoryChanged(parentUuid, userId, error, isPrivate, parentUuid == null, UPDATE_DIRECTORY);
```

Comment:
See previous comment about use of NotificationType.UPDATE_DIRECTORY

Reply:
Done